### DISCUSSION TITLE
RNAV Approaches

### DISCUSSION THREAD

**Discussion by Daboj1337** [flybywiresim/aircraft#3454] [Q&A] [answered]
Is it possible at the moment to fly rnav approaches with the right symbols on the PFD?

**Comment by robinottawa** [comment]
According to the 320 sim pilot on youtube, it is. (Not sure about the right symbols.)

(But for me, there are always things going wrong and I end up flying by hand, and I can't tell if it's me or the sim.)

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
RNAV approaches are not working properly atm, as RNAV approaches are dependent on VNAV which the a32nx doesn't have. So it is best to stick with loc or ils approaches for now.
